frontend/src/components/account/Register.jsx

```diff
@@ -43,8 +47,6 @@ export default function Register() {
 
   // decide which step is actively showing
   const [activeStep, setActiveStep] = useState(0);
```

Comment:
I think this was already like this previously but it might be better to make activeStep correspond with an enum or const variable with the name of the step to improve readability.

Reply:
Resolved! Using const variables now to improve readability.